packages/rln/src/contract/rln_contract.ts

```diff
@@ -152,7 +152,7 @@ export class RLNContract {
    */
   public async getMinRateLimit(): Promise<number> {
     const minRate = await this.contract.minMembershipRateLimit();
-    return minRate.toNumber();
+    return ethers.BigNumber.from(minRate).toNumber();
```

Comment:
Why this change I am curious?

Reply:
similar to the prev PR, contract returning BigNumber needs to be converted